### PR TITLE
fix(sr): dcm4chee requires the patient name for an SR to match what is in the original study

### LINKED ISSUE #3648: [Bug] Exporting measurements in DICOM SR results in DICOM Store (409 Conflict) issue
### Describe the Bug

Envrionments use:
1.Using. DCM4Chee docker installation [https://docs.ohif.org/configuration/dataSources/dicom-web#open-source-dicom-image-archives]
2. Using master branch of OHIF Viewer
3. Running yarn run dev:dcm4chee

Exporting measurements as DICOM SR in a local environment do not go through. 409 Conflict is raised when saving the DICOM SR file back to DCM4Cher server. 

### Steps to Reproduce

1. Upload any study
2. Make a measurement
3. Click yes to "Track measurements for this series?"
4. Click Create Report button in right hand panel
5. Add name to saved report. 
6. Click Save

open console in Chrome. 

Notice the following error:
catcher.js:197     POST http://localhost:8080/dcm4chee-arc/aets/DCM4CHEE/rs/studies 409 (Conflict)

### The current behavior

Create report functionality is not working in local environment. 

### The expected behavior

Create report functionality should work. 

### OS

macOS Venture 13.5

### Node version

v18.10.0

### Browser

Chrome v116.0.5845.179

### PR DESCRIPTION
**Pull request by jbocce**
### Context

Fixes #3648

### Changes & Results

Format the patient name of a measurement report as per DICOM.

### Testing

See #3648

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] OS: Windows 11
- [x] Node version: 18.16.1
- [x] Browser: Chrome 118.0.5993.71
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
is this related to this too? https://github.com/dcmjs-org/dcmjs/pull/364

**Comment by jbocce** [comment]
> is this related to this too? [dcmjs-org/dcmjs#364](https://github.com/dcmjs-org/dcmjs/pull/364)

Yes I would say just from the [second comment](https://github.com/dcmjs-org/dcmjs/pull/364#issuecomment-1692073906) it is related.

**Comment by sedghi** [comment]
@jbocce I mean does it makes sense to wait for dcmjs to update and we just consume a newer version?

**Comment by jbocce** [comment]
> @jbocce I mean does it makes sense to wait for dcmjs to update and we just consume a newer version?

Yes I think that makes sense. However there is the slice thickness change in this PR that is still needed AND we should leave this PR open just in case the timing of the dcmjs fix is not favourable for any OHIF release. Does that make sense?

File: platform/docs/docs/configuration/dataSources/dicom-json.md [hunk 1]
```diff
@@ -154,6 +154,18 @@ Your public folder should look like this:
 
 ![](../../assets/img/dicom-json-public.png)
 
+:::tip
+It is important to URL encode the `url` query parameter especially if the `url`
```
Comment (jbocce) [inline platform/docs/docs/configuration/dataSources/dicom-json.md:158]: I added these docs for DICOM JSON for an unrelated issue that I recently closed.

**Review by sedghi — APPROVED** [review]

**Pull request merged**